test_cases/tests.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+
+class TestCreateRoom(unittest.TestCase):
+	"""Test room creation """
+
+	def setUp(self):
+		self.office_space = OfficeSpace()
+		self.living_space = LivingSpace()
+
+
+	def test_officespace_creation(self):
+		initial_count_offices = len(self.office_space.total)
+		self.office_space.create_new("Office 1")
+		new_count_offices = len(self.office_space.total) 
+		self.assertEqual(new_count_offices - initial_count_offices, 1, 
+			msg = 'No new office created')
+	
+
+	def test_officespace_size_type(self):
+		self.assertEqual(self.office_space.size , 6 ,
+			msg ='Maximum capacity is 6 occupants')
+
+		self.assertEqual(self.office_space.room_type, "OFFICE",
+			msg = "Room created should be of type OFFICE")
+
+
+	def test_officespace_accepts_str(self):
+		self.assertRaises(ValueError, self.office_space.create_new, 111)
```

Comment:
Your function name is descriptive but, it does not match its contents in my opinion. I think `test_officespace_rejects_int` would be better.